src/Controllers/DriftController.php

```
@@ -239,4 +239,62 @@ public function reviewChange($id): void
+    /**
+     * Show the strategic alignment history for a project.
+     *
+     * Displays all drift alerts of type 'alignment' for the project,
+     * newest first.
+     */
+    public function alignmentHistory(): void
+    {
+        $user      = $this->auth->user();
+        $orgId     = (int) $user['org_id'];
+        $projectId = (int) $this->request->get('project_id', 0);
+        $project = ProjectPolicy::findViewableProject($this->db, $user, $projectId);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Security\ProjectPolicy' in method 'alignmentHistory'. (undefined)

(StaticAccess)